src/widgets/mentors/ui/mentors.tsx

```diff
@@ -1,19 +1,22 @@
+import classNames from 'classnames/bind';
 import { LINKS } from '@/app/const';
 import mentorImg from '@/shared/assets/mentors-wanted.webp';
 import { Image } from '@/shared/ui/image';
 import { LinkCustom } from '@/shared/ui/link-custom';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './mentors.scss';
+import styles from './mentors.module.scss';
+
+const cx = classNames.bind(styles);
 
 export const Mentors = () => {
   return (
-    <section className="mentors container" id="mentors-wanted">
-      <div className="mentors content column-2">
-        <div className="mentors-info">
+    <section className={cx('container', 'mentors-container')} id="mentors-wanted">
```

Comment:
```suggestion
    <section className={cx('container', 'mentors-container')} id="mentors-wanted" data-testid="mentors-wanted">
```
Also use constant for `id` - please, add it to `src/app/const/index.ts`

Reply:
I don't see any constants for IDs in `src/app/const/index.ts` and I don't see the use of constants for id in the entire project. Should I create this first? Did I understand correctly, something like this:
`
const IDS = {
  MENTORS_WANTED: 'mentors-wanted'
};
`

Reply:
Done